README.md

```diff
@@ -150,14 +150,19 @@ Run shell commands with -y so the user doesn't have to confirm them.
 print(interpreter.system_message)
 ```
 
-### Change the Model
-
+### Change the Model - We support 100+ LLMs
+open-interpreter uses [LiteLLM](https://github.com/BerriAI/litellm) for LLM Input/Ouput
 For `gpt-3.5-turbo`, use fast mode:
 
 ```shell
 interpreter --fast
 ```
```

Comment:
can we standardize model selection for gpt-3.5 to be 

```
interpreter --model gpt-3.5-turbo
```

Reply:
Agreed. `--fast` should be deprecated.

But I don't particularly like `--model` being used for API access. I'd prefer `--local --model` to be used for local models and something like `--api` to be used to specify an API like OpenAI GPT-4 or Claude-2. It makes it more clear whether you're sending your data to the cloud or not.

Just my two cents.

Reply:
i agree i like --api too !

Reply:
i think `api` is less clear than `model`, i think choosing your makes sense! i love `--model claude-2`.

agreed it's important to make the distinction very clear in our messaging though. `--local` should always be on for local models, if off, we're on the cloud (which is the way OI is intended to be used / default behavior).

and agreed `--fast` should probably be deprecated (unless we decide it's a nice shortcut to quickly use gpt-3.5 (i use it this way) + can think of other shortcuts that are similar, so this isn't a weird one-off.)

(local _was_ like this in that it autofilled model as code-llama, but now i think it's better as a bool re: we'll attempt to run this model locally vs in the cloud)

Reply:
How about giving users a way to choose? When the user directly enters `interpreter`, a list of models is listed for the user to choose.

Reply:
I completely understand where you're coming from, and I genuinely agree with your perspective. And, I feel that having a list might ease the mental load for our users and simplify our code, essentially reducing the entry points for selecting a model. Your method seems quite feasible, and if those configurations remain confidential, I believe there shouldn't be any issues. Just sharing my thoughts.

Reply:
> and agreed `--fast` should probably be deprecated (unless we decide it's a nice shortcut to quickly use gpt-3.5 (i use it this way) + can think of other shortcuts that are similar, so this isn't a weird one-off.)

Fwiw, I also use --fast because it's simple, less friction and makes things that bit faster.

Reply:
will address that in this PR:

- [ ] interpreter --models shows a list of supported llms that a user can select 
- [ ] If a user has entered keys `interpreter --models` should show llms supported for their validated keys (no security issues)
- [ ] If a user has not entered any keys, `interpreter --models` should show 5 llms from popular providers - `anthropic/claude-2`, `anthropic/claude-1.2`, `cohere/command-nightly`, `togetherai/llama2`, `ai21/j2-ultra`

Reply:
@KillianLucas open questions:
- What should go in the config file ? 
- Should I address parsing the code generated by claude-2 in this PR ? (similar to how you're handling local llms)